train_hierarchialXC.py

```diff
@@ -58,6 +69,7 @@ def generate_encodings(data_loader, ae):
 # parse the arguments
 args = parser.parse_args()
 
+print(" a ad ", args.save_model)
```

Comment:
Heh

Reply:
oops. can i force push :) ?

Reply:
Push changes with the removal of the squeeze method